org.osgi.test.junit5/src/main/java/org/osgi/test/common/annotation/package-info.java

```diff
@@ -17,7 +17,7 @@
  *******************************************************************************/
 
 @Export(attribute = "junit=5", substitution = Substitution.NOIMPORT)
-@Version("1.1.1")
+@Version("1.2.0")
```

Comment:
Perhaps it would be good documentation to add `@since 1.2` javadoc comments to the new members/type introduced here?